logistics/scripts/modules/database/postgres-database-management.psm1

```diff
@@ -17,6 +18,20 @@ $script:packageName = "PostgreSQL.Binaries"
 $script:packageVersion = "12.2.0"
 
 function Test-PostgreSQLBinariesInstalled {
+    if (!(Get-IsWindows)) {
+        try {
+            psql -V
+        }
+        catch {
+            Write-Host "ERROR:  Postgres client binaries are not installed on this Unix system." -ForegroundColor Red
+            Write-Host "To install them you can run:" -ForegroundColor Red
+            Write-Host "    Ubuntu: apt-get install postgresql-client" -ForegroundColor Red
+            Write-Host "    Alpine: apk add postgresql-client" -ForegroundColor Red
+            return
```

Comment:
I agree with using `Write-Host` with foreground red, rather than `Write-Error`: the latter would actually raise an exception and cause execution to stop. 

However, need to return false here:

```suggestion
            return $false
```

Reply:
nice catch, fixed this